Explorer/Assets/DCL/AvatarRendering/Emotes/Helpers/MemoryEmotesCache.cs

```diff
@@ -17,13 +17,25 @@ public class MemoryEmotesCache : IEmoteCache
 
         public bool TryGetEmote(URN urn, out IEmote emote)
         {
-            if (!emotes.TryGetValue(urn, out emote)) return false;
-            UpdateListedCachePriority(@for: urn);
+            if (!emotes.TryGetValue(urn, out emote))
+            {
+                if (!emotes.TryGetValue(urn.ToLower(), out emote))
+                    return false;
+
+                urn = urn.ToLower();
```

Comment:
since `ToLower()` may be costly, we could have the `urn = urn.ToLower();` first and then just use `if (!emotes.TryGetValue(urn, out emote))` again, to avoid running the `ToLower()` twice

Reply:
Do you need to check the 'noLower' version? Since the `Set()` method below always sets to lower, I think it would be enough to do `urn = urn.ToLower()` and then use it for the rest of the method logic. That would remove one of the ifs making the lofig cleaner

Reply:
I wanted to avoid lowering when there is no no need to do so. Most of the urns comes in lower case, only few have caps. What do you think?